.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -81,8 +81,7 @@ const uploadPipeline = (pipelineContent: string | object) => {
     }
 
     if (
-      // Failing: See https://github.com/elastic/kibana/issues/177084
-      // (await doAnyChangesMatch([/^x-pack\/plugins\/apm/, /^packages\/kbn-apm-synthtrace/])) ||
+      (await doAnyChangesMatch([/^x-pack\/plugins\/apm/, /^packages\/kbn-apm-synthtrace/])) ||
```

Comment:
This path is wrong now, shoudn't be `/^x-pack\/plugins\/observability_solution\/apm/`?